dev/arktype.io/docs/api/validatebound.md

```diff
@@ -10,14 +10,22 @@ hide_table_of_contents: true
 
 ## string
 
--   "[value?][comparitor?]type[comparitor][value]"
+-   "N&lt;S&lt;N", with comparators restricted to &lt; or &lt;= <br/>
+-   "2&lt;=number&lt;5"<br/>
+-   "string[]===5"<br/>
 
-## comparitors
+## param
 
--   [&lt;,&gt;,&lt;=,&gt;=,==]
+| Variable | Description                                  |
```

Comment:
It's nice that this ends up in a formatted table like this but we have to use something other than `param` so we're not lying. Is there a way to just include a MD  table like this directly in a "description" tag?